flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/statemachine/StateMachineExample.java

```diff
@@ -82,19 +83,26 @@ public static void main(String[] args) throws Exception {
         final ParameterTool params = ParameterTool.fromArgs(args);
 
         // create the environment to create streams and configure execution
-        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        Configuration configuration = new Configuration();
+        final StreamExecutionEnvironment env =
```

Comment:
The environment should be created after setting the configuration, otherwise the settings may not take effect because the environment may maintain its own configuration object and this configuration is only read in the creation process.

Reply:
fixed